TEAMS_ROLES_STRUCTURES.md

```diff
@@ -162,97 +150,93 @@ Responsibilities include:
 Here you can find the full list of active Teams inside the IPFS Org. In addition to the roles in each Team formation, we have as:
 
 - `IPFS Project BDFL` - [**Juan Benet**](https://github.com/jbenet/)
-- `IPFS Project Lead` - [**David Dias**](https://github.com/daviddias/)
-- `IPFS Project Co-Lead` - [**Molly Mackinlay**](https://github.com/momack2)
+- `IPFS Project Lead` -  [**Molly Mackinlay**](https://github.com/momack2)
+- `IPFS Project Co-Lead` - **AVAILABLE**
```

Comment:
Are we going to keep Co-Lead or the "Future, Present, Past"?

Reply:
I got rid of the description for "co-lead" (since I think how we draw the line between a lead and a co-lead is always different). I don't want to be 100% prescriptive - I think there's a world in which someone steps up as co-lead who doesn't want to be a *future* lead - however that would be my default expectation.